flink-core/src/main/java/org/apache/flink/core/execution/JobStatusHook.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.executiongraph;
+package org.apache.flink.core.execution;
```

Comment:
After removing to core module, I think this interface should be marked as @PublicEvolving instead of @Internal. It is exposed to different users such as end-user or SQL-user, WDYT?

Reply:
Thanks for your comment, as we discussed offline, `JobStatusHook` itself is designed to be used internally by the engine, and there are no specific user business scenarios that require it for now, so it stays as is.